inst/templates/issue_template-bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "Open an issue in this repo"
```

Comment:
```suggestion
name: "Bug report"
```

The wording here was unclear to me that it was a bug report- I think most people with GitHub accounts who have made it this far will know this term?